doc2.rmd

```diff
@@ -0,0 +1,39 @@
+# ORCA
+
+ORCA is a platform for bioinformatics analysis. It is suited for those wishing to conduct self-serve analysis using their own existing data or provides a comprehensive bioinformatics environment for education or research, making results reproducible.
+
+ORCA provides access to a menu of validated bioinformatics software and the power to use these inside a secure and private containerized environment. These private containers are backed by larger mounted storage volumes and dedicated compute nodes to ensure sufficient resources for the demanding needs of bioinformatics analysis.
+Hundreds of bioinformatics tools from Homebrew-Science are installed in the ORCA Docker image using [Linuxbrew](http://linuxbrew.sh/), the package manager for Linux. ORCA is based on version 16.04 of Ubuntu Image.
+
+It’s under the MIT license using the Docker , an open-source platform that 
+packages up an application with all of the minimal dependencies it needs.
```

Comment:
ORCA itself is available under the MIT license. The software tools that it includes are licensed under various open source licenses.

ORCA is available as a Docker image that includes hundreds of preinstalled software packages.